### ISSUE TITLE
Unexpected pop pane after closing Zotero

### ISSUE THREAD

**Issue by ramonmi** [retorquere/zotero-better-bibtex#2932]
### Debug log ID

none

### What happened?

After I closed the Zotero window, a pop pane appeared. I tested it with BibTeX as the only plugin, but when I disabled all plugins, the issue would be gone. So it is related to this plugin probably, could you investigate it? Thanks a lot.
![Video_20240809213913-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/e8fa390a-83cc-4f9d-905d-47d89c771e94)

**Comment by retorquere** [comment]
It's not present on MacOS. I'll test on Windows in the weekend.

**Comment by retorquere** [comment]
reprodicible but only on Windows

**Comment by retorquere** [comment]
I have asked the Zotero crew. I don't know what it could be.

**Comment by ramonmi** [comment]
Yeah, I also noticed that it is not 100% but often. Thanks for your efforts.
